deploy-manage/monitor/stack-monitoring/collecting-monitoring-data-with-metricbeat.md

```diff
@@ -2,14 +2,16 @@
 navigation_title: "Collecting monitoring data with {{metricbeat}}"
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-metricbeat.html
+applies:
+  stack: ga 6.5.0
```

Comment:
we should just ignore any indicators prior to 9.x - so this should just be `all`